Documentation (#15)
nih-cfde/icc-eval-core: README.md

# CFDE ICC Evaluation Core

## Overview

The [Common Fund Data Ecosystem (CFDE)](https://commonfund.nih.gov/dataecosystem) is an effort to bring together knowledge across [Common Fund](https://commonfund.nih.gov/) programs into a cohesive resource.

> The Integration and Coordination Center (ICC) is a center within the CFDE responsible for, among other things, evaluating the activities of Common Fund programs.

Reviewer: ```suggestion
The Integration and Coordination Center (ICC) is a center within the CFDE responsible for, among other things, reporting on the impact and influence of the CFDE.
```

This repository is a place to:

> - View a high-level evaluation of Common Fund program activity (e.g. projects over time, dollars awarded, # of publications produced, etc.).

Reviewer: ```suggestion
- View a high-level dashboard of Common Fund and CFDE activities (e.g. projects over time, dollars awarded, # of publications produced, etc.).
```

> - Submit details about your Common Fund project to be included in the above.

Reviewer: ```suggestion
- Ensure your project's activities and materials are included in the above by aligning with the ingestion processes for the dashboard.
```

Author: I committed something similar to this.

- Maintain code used to coordinate the above.

> The evaluation can be viewed as a live webapp dashboard or as separate PDF reports.

Reviewer: ```suggestion
Reports are availabe as a live webapp dashboard or as PDFs that will be able to include narrative text and additional analyses.
```

Author: Throughout the docs and code comments, I'm trying to define "reports" as the printed pdfs and "pages" as the live dashboard pages. If you'd like me to use "reports" to mean either "pdf reports" or "webapp/page reports", that's okay but I'll want to change all instances of this throughout the repo for consistency (and avoid my own confusion).

Reviewer: A change of that scale was not intended. :)

Author: It won't take that long if you think trying to make those terms stick will be an uphill battle. Just want to be consistent either way.

[🖱️ The Webapp](cfde-eval.netlify.app)

[📜 PDF Reports](cfde-eval.netlify.app/reports)

## Submit your project

> We are able to get most details about Common Fund projects automatically from NIH resources, but there are some pieces of info that you must submit manually so that we have everything we need to create the evaluation.

Reviewer: ```suggestion
We gather details about Common Fund projects automatically from NIH resources, but there are some pieces of info that require manual intervention to be integrated.
```

Author: I've commited something similar to this.

> If you have a project under the Common Fund, please follow the instructions in the sections below as applicable.

Reviewer: ```suggestion
If you would like your project's materials to be included, please follow the instructions in the sections below as applicable.
```

Author: I've commited something similar to this.
